.github/workflows/main.yml

```diff
@@ -34,6 +34,9 @@ jobs:
         if: steps.client-cache.outputs.cache-hit != 'true'
         run: yarn install
 
+      - name: Populate app registry
+        run: yarn init-appregistry
+
```

Comment:
Probably don't need this anymore, right?

Reply:
We actually need that for a separate reason. This is because if the `appregistry.json` file isn't present, the linting will fail, because it can't import the file. I opted for this instead of switching `yarn build` to run before linting, because if we did that, then we wouldn't only see the build step failures, and not the informative ones from linting.